<ui version="4.0" >
 <class>ViewmdaWidget</class>
 <widget class="QMainWindow" name="ViewmdaWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ViewMda</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>33</y>
     <width>394</width>
     <height>222</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="ViewmdaWidget2D" native="1" name="widget2d" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="ViewmdaWidget1D" native="1" name="widget1d" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QScrollBar" name="scrollbar" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>394</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>255</y>
     <width>394</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>21</y>
     <width>394</width>
     <height>12</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ViewmdaWidget2D</class>
   <extends>QWidget</extends>
   <header>viewmdawidget2d.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ViewmdaWidget1D</class>
   <extends>QWidget</extends>
   <header>viewmdawidget1d.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
